config/components/wavefront.json

```
@@ -1,4 +1,5 @@
 {
   "name": "wavefront",
-  "cpeProduct": "wavefront_by_vmware_nozzle"
+  "cpeProduct": "wavefront_by_vmware_nozzle",
+  "to-be-deprecated": "20230702"
```

We are not yet in July 🍹🏖️ 
```suggestion
  "to-be-deprecated": "20230602"
```

---

The checklist says: Update JSON config in [vulndb](https://github.com/bitnami/vulndb) repo (`deprecated: <date>`) and a deprecation notice in `README.md`. Add the date in the format <yyyymmdd> (i.e. 20231231) with at least one month ahead.

---

My fault. It's expected to set this date at least one month ahead.